<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>678</width>
    <height>502</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidgets">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="tabMap">
       <attribute name="title">
        <string>Map</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGraphicsView" name="mapView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabCreature">
       <attribute name="title">
        <string>Creature</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Generic</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <layout class="QFormLayout" name="formLayout_2">
             <property name="fieldGrowthPolicy">
              <enum>QFormLayout::ExpandingFieldsGrow</enum>
             </property>
             <item row="0" column="0">
              <widget class="QLabel" name="creatureEntryLabel">
               <property name="text">
                <string>Creature Entry</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="creatureEntryLine">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="nameLabel">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="creatureNameLine"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="tilesetLabel">
               <property name="text">
                <string>Tileset</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="tilesetLineEdit"/>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="tilesetXLabel">
               <property name="text">
                <string>Tileset X</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSpinBox" name="tilesetXSpinBox"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="tilesetYLabel">
               <property name="text">
                <string>Tileset Y</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QSpinBox" name="tilesetYSpinBox"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="nPCFlagsLabel">
               <property name="text">
                <string>NPC Flags</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QCheckBox" name="isQuestGiverCheck">
                 <property name="text">
                  <string>Quest Giver</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="isVendorCheck">
                 <property name="text">
                  <string>Vendor</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="isRepairCheck">
                 <property name="text">
                  <string>Repair</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="maxHealthLabel">
               <property name="text">
                <string>Max Health</string>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QSpinBox" name="maxHealthSpinBox">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>900000000</number>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="maxPowerLabel">
               <property name="text">
                <string>Max Power</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QSpinBox" name="maxPowerSpinBox">
               <property name="maximum">
                <number>900000000</number>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="scriptNameLabel">
               <property name="text">
                <string>Script Name</string>
               </property>
              </widget>
             </item>
             <item row="9" column="1">
              <widget class="QLineEdit" name="scriptNameLineEdit"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="addCreatureToDatabase">
             <property name="text">
              <string>Add to database</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabItem">
       <attribute name="title">
        <string>Item</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Generic</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="5" column="1">
            <widget class="QComboBox" name="subClassBox"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Description</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="classBox"/>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Sub class</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Class</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QSpinBox" name="displayIDEdit">
             <property name="maximum">
              <number>900000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="nameEdit"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Inventory Type</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Display ID</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="nameDescription"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Name</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="invTypeBox">
             <item>
              <property name="text">
               <string>Not equippable</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Bag</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="9" column="1">
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>ItemID</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="itemID_Line">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="bag_specific_group">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="title">
           <string>Bag-specific</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Container slots:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="containerSlotsBox">
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>36</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QPushButton" name="addItemToDatabase">
          <property name="text">
           <string>Add to database</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabQuest">
       <attribute name="title">
        <string>Quest</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="questIDLabel">
            <property name="text">
             <string>Quest ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="questIDLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="titleLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="detailsLabel">
            <property name="text">
             <string>Details</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="objectiveLabel">
            <property name="text">
             <string>Objective</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPlainTextEdit" name="plainTextEdit_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addQuestToDatabase">
          <property name="text">
           <string>Add to database</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabDisplayID">
       <attribute name="title">
        <string>DisplayIDs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="displayInfoIDLabel">
            <property name="text">
             <string>DisplayInfoID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="displayInfoID_SpinBox"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="textureFilenameLabel">
            <property name="text">
             <string>Texture Filename</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="textureFilenameEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="addDisplayIDToDatabase">
          <property name="text">
           <string>Add to database</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabHelp">
       <attribute name="title">
        <string>Help</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPlainTextEdit" name="helpEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="plainText">
           <string>First,
you need to open a &quot;data connection&quot;

Open a server database (MySQL)

For merge all data.
Use Utilities -&gt; Merge server db to client file
</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>678</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open ...</string>
     </property>
     <addaction name="actionA_server_database"/>
     <addaction name="actionA_map"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="actionCreate_map"/>
    <addaction name="actionSave_map"/>
   </widget>
   <widget class="QMenu" name="menuUtilities">
    <property name="title">
     <string>Utilities</string>
    </property>
    <addaction name="actionMerge_server_database_to_client_database"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuUtilities"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionA_server_database">
   <property name="text">
    <string>a server database</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionA_map">
   <property name="text">
    <string>a map</string>
   </property>
  </action>
  <action name="actionMerge_server_database_to_client_database">
   <property name="text">
    <string>Merge server database to client database</string>
   </property>
  </action>
  <action name="actionCreate_map">
   <property name="text">
    <string>Create map</string>
   </property>
  </action>
  <action name="actionSave_map">
   <property name="text">
    <string>Save map</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
